src/extra/Changelog/helpers.js

```diff
@@ -0,0 +1,25 @@
+export default (data) => {
+	const untilDate = new Date(new Date().setDate(new Date().getDate() - 90));
+	const dateStringsIterator =
+		data.matchAll(
+			/\n# v\d+\.\d+\.\d+\n## \(([0-9]{4}-[0-1][0-9]-[0-3][0-9])\)/g
```

Comment:
You could use multiline mode + `^` instead:
```suggestion
			/^# v\d+\.\d+\.\d+\n## \(([0-9]{4}-[0-1][0-9]-[0-3][0-9])\)/gm
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#advanced_searching_with_flags